dist/index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <link rel="stylesheet" href="stylesheet.css">
-  <link href="https://fonts.googleapis.com/css?family=Noto+Sans+TC:300,400,700&display=swap" rel="stylesheet">
-  <title>ReviewsAndDrawers</title>
-</head>
-<body>
-  <div id="root"></div>
-  <script type='text/javascript' src="bundle.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="stylesheet.css" />
```

Comment:
You might want to add `type="text/css"` to your stylesheet link.  It appears to be working without it, but that's standard.